### PR TITLE
Spring Boot starter release blog post

### PR DESCRIPTION
**Pull request by karesti**
We have released the starter with 9.4.17.Final and 10.1.1.Final

### CODE REVIEW

File: blog/2020-02-10-spring-boot-release-2.2.x.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Infinispan in client/server mode has made important changes on https://infinispan.org/docs/stable/titles/developing/developing.html#marshalling[marshalling].
+Previous versions of Infinispan used JBoss Marshaller as a default.
+Now https://github.com/infinispan/protostream[ProtoStream] that works with https://developers.google.com/protocol-buffers[Protocol Buffers] is
+the default Marshaller.
+
+Unfortunately ProtoStream does not work yet with our Spring Cache and Session support.
+This might be supported in the future versions, but some important changes are required to do so in the actual implementation.
+
+Spring users in client/server mode, we need to configure two additional properties today:
```
Comment (pruivo) [inline blog/2020-02-10-spring-boot-release-2.2.x.adoc]: `we need` => `they need`?

File: blog/2020-02-10-spring-boot-release-2.2.x.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+`infinispan.remote.marshaller=org.infinispan.commons.marshall.JavaSerializationMarshaller`
+`infinispan.remote.java-serial-whitelist=org.infinispan.tutorial.simple.spring.remote.*`
```
Comment (pruivo) [inline blog/2020-02-10-spring-boot-release-2.2.x.adoc]: why the simple tutorials? 
may I suggest use: 
`my.package.MyClass1,my.package.MyClass2,...`
?

File: blog/2020-02-10-spring-boot-release-2.2.x.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+In the first property we specify that we want to use the
+ https://infinispan.org/docs/stable/titles/developing/developing.html#java_serialization_marshaller[Java Serialization Marshaller].
+
+The second property is to specify which classes, or packages, separated with `,`, need to be marshalled
+by the java serialization.
+
+Spring users can also keep using the https://infinispan.org/docs/stable/titles/developing/developing.html#jboss_marshalling[JBoss Marshalling]
+if we want to.
```
Comment (pruivo) [inline blog/2020-02-10-spring-boot-release-2.2.x.adoc]: I see `we` everywhere. Initially, it sounds weird but, to be honest, I'm not sure what is the correct pronoun to use... I'll let you decide that :)

File: blog/2020-02-10-spring-boot-release-2.2.x.adoc (hunk 1 above)
Comment (karesti) [reply blog/2020-02-10-spring-boot-release-2.2.x.adoc]: @pruivo Don has done his Don touch

File: blog/2020-02-10-spring-boot-release-2.2.x.adoc (hunk 2 above)
Comment (karesti) [reply blog/2020-02-10-spring-boot-release-2.2.x.adoc]: I copied the simple tutorials of infinispan sb classes

**Comment by tristantarrant** [comment]
Merged

**Pull request merged**